triaging.rst

```diff
@@ -375,6 +375,13 @@ Various informational flags about the issue. Multiple values are possible.
 | easy          | Fixing the issue should not take longer than a day for     |
 |               | someone new to contributing to Python to solve.            |
 +---------------+------------------------------------------------------------+
+| easy (C)      | Fixing the issue should not take longer than a day for     |
+|               | someone new contributing to Python, focused on C/C++.      |
++---------------+------------------------------------------------------------+
+| security_issue| The issue would fit as, or is related as a security issue. |
```

Comment:
This should clarify the relationship with the "security" issue type: Why do they both exist? What's the difference? When should one use this or the other, or both?

I don't know the answers to these; some research will likely be required.

Reply:
From what I see, it looks like "security" alone might have a security issue while "security_issue" is a definite issue that needs a fix, I don't believe you can use both since the later is an escalation of just "security".